__tests__/server_test.js

```diff
@@ -42,16 +42,15 @@ describe("Server endpoint testing", function() {
       })
     })
 
-    it("returns an empty array on a GET request on '/events/somebadevent'", function (done) {
+    it("returns an 404 on an incorrect GET request on '/events/somebadevent'", function (done) {
       request(app).get('/events/somebadevent').then((response) => {
-        expect(response.statusCode).toBe(200);
-        expect(response.body).toEqual([]);
+        expect(response.statusCode).toBe(404);
```

Comment:
I though we were returning just an empty array in this case?

Reply:
If you are hitting a wrong endpoint, I thought we were returning a 404 because that endpoint wasn't correct. We are still returning an empty array for when no responses are returning. Can change if you want, but thought this made more sense.